dev/benchmarks/macrobenchmarks/lib/src/web/bench_platform_view_infinite_scroll.dart

```diff
@@ -0,0 +1,118 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:html' as html;
+
+import 'package:flutter/material.dart';
+
+// TODO(mdebbar): flutter/flutter#55000 Remove this conditional import once
+// web-only dart:ui_web APIs are exposed from a dedicated place.
+import 'platform_views/non_web.dart'
+    if (dart.library.html) 'platform_views/web.dart';
+import 'recorder.dart';
+
+const String benchmarkViewType = 'benchmark_element';
+
+void _registerFactory() {
+  platformViewRegistry.registerViewFactory(benchmarkViewType, (int viewId) {
+    final html.Element htmlElement = html.DivElement();
```

Comment:
Nit: maybe worthwhile to incorporate the `viewId` into the `id` of the html element?